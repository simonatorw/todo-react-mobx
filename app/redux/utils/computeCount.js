export function computeCount(list) {
    return list.length;
}