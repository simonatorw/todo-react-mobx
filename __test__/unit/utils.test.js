import { computeCount } from '../../app/redux/utils/computeCount';

describe('Todo utils tests', () => {
    test('It can compute the count of a list', () => {
        const list = [0, 1];
		
        expect(computeCount(list)).toBe(2);
    });
	
	test('It will match snap shot', () => {
        const list = [0, 1];
		
        expect(computeCount(list)).toMatchSnapshot();		
	});
});