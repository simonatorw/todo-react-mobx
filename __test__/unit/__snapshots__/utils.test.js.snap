// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo utils tests It will match snap shot 1`] = `2`;
